.github/workflows/build.yml

```diff
@@ -26,8 +26,8 @@ jobs:
             pip install -r test-requirements.txt
             pytest ./tests
 
-      - name: Build RPM (Rockylinux:8)
-        run: docker run -v `pwd`:/scylla-machine-image -w /scylla-machine-image  --rm rockylinux:8 bash -c 'dnf update -y; dnf install -y git ; git config --global --add safe.directory "*"; ./dist/redhat/build_rpm.sh -t centos8'
+      - name: Build RPM (CentOS 8)
```

Comment:
Just a general question, this is our unit-test, why to invest time moving to ubi? why does it need to be change at all?

Reply:
anyhow at the end the actual build would be with dbuild based on fedora

if we want to aligned anything here, it should be with the fedora version within dbuild

not sure how UBI is related here.

Reply:
yes, so unless we move dbuild (and any other containers) to be based on `ubi` i don't think we should change this. We can however use updated OS (like Rockylinux:9)

Reply:
We are just verifying that nothing is breaking our package build process

Reply:
But we are not using our build process, but another process, that we have implemented in this script... 
Confusing.

Reply:
Since the build process of this one is spread across multiple repos

This was introduced at a unit test of the part that exists in this repository.

We are trying to tell you that from the start...